tests/conftest.py

```diff
@@ -12,8 +12,25 @@ def generated_rows(request):
     def row_values(index, value):
         return mockobj.mock_calls[index][1][1][value]
 
+    def table_values(tablename, index, field=None):
+        "Look up a value from a table."
```

Comment:
```suggestion
        """Look up a value from a table."""
```

(I know it's equivalent, but the visual consistency in how docstrings appear is helpful when scanning through code.)

Reply:
Fixed in f3b2629